diff -urN audiocd-kio-4.10.4/plugins/vorbis/encodervorbis.cpp audiocd-kio-4.10.4-patched/plugins/vorbis/encodervorbis.cpp
--- audiocd-kio-4.10.4/plugins/vorbis/encodervorbis.cpp	2013-05-29 06:31:08.000000000 +1100
+++ audiocd-kio-4.10.4-patched/plugins/vorbis/encodervorbis.cpp	2013-06-12 01:38:14.317582560 +1100
@@ -25,8 +25,6 @@
 #include "encodervorbis.h"
 #include "audiocd_vorbis_encoder.h"
 
-#ifdef HAVE_VORBIS
-
 #include <vorbis/vorbisenc.h>
 #include <time.h>
 #include <stdlib.h>
@@ -139,11 +137,8 @@
   // Now that we have read in the settings apply them to the encoder lib
   switch (d->vorbis_encode_method) {
        case 0:
-/* Support very old libvorbis by simply falling through.  */
-#if HAVE_VORBIS >= 2
        vorbis_encode_init_vbr(&d->vi, 2, 44100, d->vorbis_quality/10.0);
        break;
-#endif
        case 1:
        vorbis_encode_init(&d->vi, 2, 44100, d->vorbis_bitrate_upper, d->vorbis_bitrate_nominal, d->vorbis_bitrate_lower);
        break;
@@ -155,18 +150,11 @@
   long processed(0);
 
   while(vorbis_analysis_blockout(&d->vd,&d->vb)==1) {
-    /* Support ancient libvorbis (< RC3).  */
-#if HAVE_VORBIS >= 2
     vorbis_analysis(&d->vb,NULL);
     /* Non-ancient case.  */
     vorbis_bitrate_addblock(&d->vb);
 
     while(vorbis_bitrate_flushpacket(&d->vd, &d->op)) {
-#else
-    vorbis_analysis(&d->vb,&d->op);
-    /* Make a lexical block to place the #ifdef's nearby.  */
-    if (1) {
-#endif
       ogg_stream_packetin(&d->os,&d->op);
       while(int result=ogg_stream_pageout(&d->os,&d->og)) {
         if (!result) break;
@@ -193,9 +181,6 @@
   switch (d->vorbis_encode_method)
   {
   case 0: // quality based encoding
-
-#if HAVE_VORBIS >= 2 // If really old Vorbis is being used, skip this nicely.
-
   {
     // Estimated numbers based on the Vorbis FAQ:
     // http://www.xiph.org/archives/vorbis-faq/200203/0030.html
@@ -209,9 +194,6 @@
 
     break;
   }
-
-#endif // HAVE_VORBIS >= 2
-
   default: // bitrate based encoding
     vorbis_size = (time_secs * d->vorbis_bitrate/8);
     break;
@@ -321,5 +303,3 @@
   }
 }
 
-#endif // HAVE_VORBIS
-
diff -urN audiocd-kio-4.10.4/plugins/vorbis/encodervorbis.h audiocd-kio-4.10.4-patched/plugins/vorbis/encodervorbis.h
--- audiocd-kio-4.10.4/plugins/vorbis/encodervorbis.h	2013-05-29 06:31:08.000000000 +1100
+++ audiocd-kio-4.10.4-patched/plugins/vorbis/encodervorbis.h	2013-06-12 01:38:14.317582560 +1100
@@ -26,7 +26,6 @@
 #define ENCODER_VORBIS_H
 
 #include "ui_encodervorbisconfig.h"
-#ifdef HAVE_VORBIS
 
 #include <audiocdencoder.h>
 
@@ -41,7 +40,6 @@
 
 /**
  * Ogg Vorbis encoder.
- * This encoder is only enabled when HAVE_VORBIS is set.
  * Check out http://www.vorbis.com/ for lots of information.
  */
 class EncoderVorbis : public AudioCDEncoder {
@@ -70,7 +68,5 @@
 
 };
 
-#endif // HAVE_VORBIS
-
 #endif // ENCODER_VORBIS_H
 
